src/legacy/core_plugins/kibana/public/discover/np_ready/angular/discover.js

```diff
@@ -862,11 +889,9 @@ function discoverController(
     });
   }
 
-  function logInspectorResponse(resp) {
-    inspectorRequest.stats(getResponseInspectorStats($scope.searchSource, resp)).ok({ json: resp });
-  }
-
   $scope.updateTime = function() {
+    $scope.timeRangeObj = timefilter.getTime();
```

Comment:
I see no one consuming this variable

Reply:
thx for detecting this zombie, no longer lurks around